commands/issue.go

```diff
@@ -39,7 +39,15 @@ var (
 `,
 	}
 
+	cmdCreateIssue = &Command{
+		Key:   "create",
+		Run:   createIssue,
+		Usage: "issue create [-m <MESSAGE>|-f <FILE>] [-l <LABELS>]",
```

Comment:
I believe Usage and Long of subcommand are never used. Actually putting something in there can be misleading.